source/docs/03-metadata.xml

```diff
@@ -1382,156 +1354,308 @@
+                        <p>The genetic aspect of applying patches or cutting out parts of a page is described in <ptr target="#genetic"/>.</p>
+                     </div>
+                  </div>
+                  <div xml:id="layoutDescSupportDesc" type="div4">
+                     <head>Description of Layout and Support</head>
+                     <p></p>
+                     <p>
+                        <specList>
+                           <specDesc key="layoutDesc"/>
+                           <specDesc key="supportDesc"/>
+                        </specList>
+                        </p>
+                  <p></p>
+                  </div>
+                  <div xml:id="scriptDesc" type="div4">
+                     <head>Description of Script</head>
+                     <p></p>
+                     <p>
+                        <specList>
+                           <specDesc key="scriptDesc"/>
+                        </specList>
+                     </p>
+                     <p></p>
+                  </div>
+               <div xml:id="sourcesHandwirtten" type="div4">
+                     <head>Additional for describing handwritten sources</head>
+                     <p></p>
+                     <p>
+                        <specList>
+                           <specDesc key="handList"/>
+                           <specDesc key="inscription"/>
+                           <specDesc key="explicit"/>
+                        </specList>
+                     </p>
+                     <p></p>
                   </div>
-                  <div xml:id="cutouts" type="div4">
-                     <head>Cutouts</head>
+                  <div xml:id="sourcesPrinted" type="div4">
+                     <head>Additional for describing printed sources</head>
```

Comment:
```suggestion
                     <head>Additional elements for describing printed sources</head>
```

Reply:
for Describing Printed Sources (capitalization)

Reply:
I'll fix that separately.